README.md

```diff
@@ -0,0 +1,80 @@
+Mapcrafter snap package
```

Comment:
Can you maybe create a top-level readme file where you reference the readme file specific for the snap packages (in the snap directory) or have the snap package documentation as a subsection of the top-level readme? Just so that it doesn't look like this repo is just for the snap package. Thanks.